models/user/user.go

```diff
@@ -827,7 +827,43 @@ func validateUser(u *User) error {
 	return ValidateEmail(u.Email)
 }
 
+// updateUserAllowed is used to block updating selected user fields when local user managemement is disabled.
+func updateUserAllowed(u *User) error {
+	// Don't allow changes of selected user fields if local user management is disabled.
+	if setting.Service.DisableLocalUserManagement && (u.Type == UserTypeIndividual) {
```

Comment:
The linter doesn't like unnecessary parentheses(issue is also present in other places in this PR)

```suggestion
	if setting.Service.DisableLocalUserManagement && u.Type == UserTypeIndividual {
```

Reply:
Done in 25673a937efd78833756f298bfd344b214158151.